app.rb

```diff
@@ -19,14 +19,15 @@ def initialize
 
   def call(env)
     lines = if LOG_REQUEST_URI
-      [env['REQUEST_URI']]
-    else
-      HerokuLogParser.parse(env['rack.input'].read).collect {|m| m[:message] }
-    end
+              [{ msg: env['REQUEST_URI'], ts: '' }]
+            else
+              HerokuLogParser.parse(env['rack.input'].read).collect { |m| { msg: m[:message], ts: m[:emitted_at].strftime('%Y-%m-%dT%H:%M:%S.%L%z') } }
+            end
```

Comment:
I’d suggest to fixup and use 2 space indentation

Reply:
Fixed